src/bootstrap/builder.rs

```diff
@@ -860,28 +860,16 @@ impl<'a> Builder<'a> {
             stage = compiler.stage;
         }
 
-        let mut extra_args = String::new();
+        let mut rustflags = Rustflags::new();
+        rustflags.env(&format!("RUSTFLAGS_STAGE_{}", stage));
```

Comment:
This looks like new behavior -- we didn't have stage-specific rustflags previously? I'd prefer to not have them since in theory stage 1 and 2 should be equivalent from a flags perspective

Reply:
Ah yeah sorry this was probably a bad rebase, it's gone in the final diff though!